v0.3/cruorg/components/list/list.scss

```diff
@@ -2,3 +2,18 @@
 @import '../../branding-imports';
 
 /* List styles */
+
+@mixin cru-list-extra-spacing {
+    margin-bottom: $cru-spacer;
+}
+
+.list {
+    .cmp-list__item {
+        @include cru-text;
+        font-size: $cru-font-size-base;
+        padding-left: $cru-spacer-base * 1.5;
+    }
+    &.cru-list-extra-spacing .cmp-list__item {
+        @include cru-list-extra-spacing;
+    }
+}
```

Comment:
@sebamelzi Please add an extra line to the end of this file.

Reply:
@sebamelzi Go ahead and give the links a dark gray color for now and leave the underline.

Reply:
@sebamelzi More single-use mixins. Please put the CSS rules in its rightful place.